aemedge/blocks/offer-cards/offer-cards.css

```diff
@@ -0,0 +1,217 @@
+
+.offer-cards-content {
+  color: var(--clr-white);
+  border: none;
+  display: flex;
+  -webkit-box-pack: justify;
+  flex-flow: column;
+  inset: 0;
+  background: linear-gradient(#33334c00 50%, 85%, #171725 100%), radial-gradient(110% 100% at 75% 25%, #20203200 0%, #171725 100%);
+  box-sizing: border-box;
+  width: 100%;
+  -webkit-box-align: start;
+  align-items: start;
+  text-align: left;
+  position: relative;
+  height: 100%;
+    padding: 28px;
+  }
+
+.offer-cards-content .text-cta-container {
+    margin: 1rem 0;
+}
+
+.offer-cards-content .headline p {
+    font-size: var(--heading-font-size-l);
+    line-height: 30px;
+}
+
+  .offer-cards-content h1 {
+    font-size: 30px;
+    line-height: 1.2;
+    font-weight: 600;
+    letter-spacing: -1.15px;
+    margin: 0 0 1rem;
+    max-width: 600px;
+  }
+
+  .offer-cards-content h2 {
+    font-size: var(--heading-font-size-3xl);
+    line-height: 54px;
+    letter-spacing: -1.15px;
+    margin: 0 0 1rem;
+    max-width: 600px;
+    -webkit-font-smoothing: antialiased;
+    text-wrap: wrap;
+    scroll-margin: 0;
+  }
+
+  .offer-cards .background {
+    position:absolute;
+    z-index: 0;
+    inset: 0;
+    height: 100%;
+
+      picture {
+          display: unset;
+      }
+  }
+
+  .offer-cards .background picture img{
+    position: relative;
+    z-index: 0;
+    object-fit: cover;
+    width: 100%;
+    box-sizing: border-box;
+    inset: 0;
+      height: 100%;
+  }
+
+  .offer-cards-content .sub-headline picture img {
+    width: auto;
+    height: auto;
+  }
+
+  .offer-cards-content .text-cta-container .button-container
+  {
+    display: flex;
+    flex-direction: column;
+    -webkit-box-align: center;
+    align-items: center;
+    width: fit-content;
+    max-width: unset;
+  }
+
+  .offer-cards-content .text-cta-container .button-container p {
+  margin:0;
+ }
+
+
+.offer-cards-content .foreground-container {
+  margin: 1rem auto;
+    text-align: center;
+}
+
+.offer-cards-content .badge {
+    line-height: 1;
+    vertical-align: bottom;
+    padding: 0 7px;
+    font-size: 0.875rem;
+    font-weight: 600;
+    background-color: rgb(16 6 159);
+    color: rgb(245 245 246);
+    border-radius: 0.125rem;
+    text-transform: uppercase;
+    margin: 0 0 1rem;
+    width: fit-content;
+}
+
+.offer-cards-container .default-content-wrapper .button-container {
+    margin-left: 114px;
+}
+
+.offer-cards-wrapper {
+    margin-bottom: 20px;
+}
+
+@media (width >= 768px) {
+    .offer-cards.block {
+        min-height: 566px;
+        box-shadow: rgb(7 7 11 / 40%) 0 4px 8px 0;
+        min-width: 23.25rem;
+        width: 372px;
+    }
+
+    .offer-cards-content {
+        min-height: 566px;
+        padding: 20px;
+    }
+
+    .offer-cards .background picture img{
+        height: 566px;
+    }
+    
+    .section.offer-cards-container{
+        display: flex;
+        flex-wrap: wrap;
+        max-width: fit-content;
+        margin-left: auto;
+        margin-right: auto;
```

Comment:
add this after line 146:
`justify-content: center;`

Reply:
You are making good progress on the layout. The component demo page says "Image size for all devices: 373 x 224 px", so yours is too large. For mobile, you still need to hide the bullets.

![image](https://github.com/user-attachments/assets/220bd09b-8642-4278-90a8-f7fe5af37bbe)